demo/test/components/previews/primer/docs/layout_preview.rb

```diff
@@ -0,0 +1,29 @@
+module Primer
+  module Docs
+    class LayoutPreview < ViewComponent::Preview
+      def default; end
```

Comment:
Wouldn't it be nice if we didn't have to define these methods? (Like controllers...)